cdisc_rules_engine/services/data_readers/dataset_json_reader.py

```diff
@@ -0,0 +1,58 @@
+import pandas as pd
+import json
+
+from cdisc_rules_engine.interfaces import (
+    DataReaderInterface,
+)
+
+# Testing only
+# test_file_path = "D:\\CDISC_CORE_Engine\\Python_work_Jozef\\lb.json"
+
+
+class DatasetJSONReader(DataReaderInterface):
+    def from_file(self, file_path):
+        # print("dataset_json_reader: starting reading_from_file = ", file_path)
+        f = open(file_path)
```

Comment:
you can read files in python by setting up a context like:
```
with open(file_path) as f:
    data = json.load(f)
```
This ensures that the file is closed once the scope is exited.

Reply:
Thanks! This is still a bit a strange construct for me, coming from the Java world ...